docs/Developers/Newcomers_Guide.rst

```diff
@@ -164,6 +165,13 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    You can view some of our repositories on
+    `GitLab <https://gitlab.com/coala>`_.
+    For more information about it, browse to our
+    `wiki page <https://github.com/coala/coala/wiki/GitLab>`_.
```

Comment:
I think it should go down one paragraph to "pick an issue" :) That fits better, so people have more options to choose issues from :)